### ISSUE TITLE
Playing videos results in "Error during loading the video."

### ISSUE THREAD

**Issue by drwb** [bpatrik/pigallery2#808]
**Describe the bug**

Playing videos has stopped working, resulting in a "Error during loading the video." 

It's been working perfectly for months.

![SCR-20240106-jwr](https://github.com/bpatrik/pigallery2/assets/16479230/5dff2e23-bc4b-44a6-ab05-0697288ef943)

**Photo/video (optional) that causes the bug**
The mediainfo for one of the videos is below, but it is repeatable for each video in each folder.
<details><summary>mediainfo  1.mp4</summary>
<p>

General
Complete name                            : 1.mp4
Format                                   : MPEG-4
Format profile                           : Base Media
Codec ID                                 : isom (isom/iso2/avc1/mp41)
File size                                : 809 MiB
Duration                                 : 8 min 2 s
Overall bit rate                         : 14.1 Mb/s
Writing application                      : Lavf57.71.100

Video
ID                                       : 2
Format                                   : AVC
Format/Info                              : Advanced Video Codec
Format profile                           : Main@L3.1
Format settings                          : CABAC / 4 Ref Frames
Format settings, CABAC                   : Yes
Format settings, Reference frames        : 4 frames
Codec ID                                 : avc1
Codec ID/Info                            : Advanced Video Coding
Duration                                 : 8 min 2 s
Bit rate                                 : 14.0 Mb/s
Width                                    : 1 080 pixels
Height                                   : 1 920 pixels
Display aspect ratio                     : 0.562
Frame rate mode                          : Constant
Frame rate                               : 30.000 FPS
Color space                              : YUV
Chroma subsampling                       : 4:2:0
Bit depth                                : 8 bits
Scan type                                : Progressive
Bits/(Pixel*Frame)                       : 0.225
Stream size                              : 799 MiB (99%)
Writing library                          : x264 core 148
Encoding settings                        : cabac=1 / ref=3 / deblock=1:0:0 / analyse=0x1:0x111 / me=hex / subme=7 / psy=1 / psy_rd=1.00:0.00 / mixed_ref=1 / me_range=16 / chroma_me=1 / trellis=1 / 8x8dct=0 / cqm=0 / deadzone=21,11 / fast_pskip=1 / chroma_qp_offset=-2 / threads=48 / lookahead_threads=8 / sliced_threads=0 / nr=0 / decimate=1 / interlaced=0 / bluray_compat=0 / constrained_intra=0 / bframes=3 / b_pyramid=2 / b_adapt=1 / b_bias=0 / direct=1 / weightb=1 / open_gop=0 / weightp=2 / keyint=90 / keyint_min=9 / scenecut=40 / intra_refresh=0 / rc_lookahead=40 / rc=abr / mbtree=1 / bitrate=14000 / ratetol=1.0 / qcomp=0.60 / qpmin=0 / qpmax=69 / qpstep=4 / ip_ratio=1.40 / aq=1:1.00
Color range                              : Limited
Color primaries                          : BT.709
Transfer characteristics                 : BT.709
Matrix coefficients                      : BT.709
Codec configuration box                  : avcC

Audio
ID                                       : 1
Format                                   : AAC LC
Format/Info                              : Advanced Audio Codec Low Complexity
Codec ID                                 : mp4a-40-2
Duration                                 : 8 min 2 s
Bit rate mode                            : Constant
Bit rate                                 : 160 kb/s
Channel(s)                               : 2 channels
Channel layout                           : L R
Sampling rate                            : 44.1 kHz
Frame rate                               : 43.066 FPS (1024 SPF)
Compression mode                         : Lossy
Stream size                              : 9.21 MiB (1%)
Default                                  : Yes
Alternate group                          : 1

</p>
</details> 

I've verified that the videos are playable in-browser natively, that they are accessible to the host and that they are mounted and accessible to the container via `docker exec`. 

I've verified the same behaviour from a brand new docker instance (virgin config, db, temp files).

**Server logs (optional)**
I can provide logs, but having run the debug log and attempted to play the video, nothing of any relevance appeared in them.

**Environment (please complete the following information):**
 - OS: bpatrik/pigallery2@sha256:f169ee70481d4bc531cab7c342e16af9e84dfba357ec3070ba9731d306c44ee6 edge-debian-bullseye on Ubuntu 22.04.3 LTS, updated nightly with watchtower.

<details><summary>docker info</summary>
<p>

Client:
 Context:    [name]
 Debug Mode: false
 Plugins:
  buildx: Docker Buildx (Docker Inc., v0.9.1)
  compose: Docker Compose (Docker Inc., v2.10.2)
  extension: Manages Docker extensions (Docker Inc., v0.2.9)
  sbom: View the packaged-based Software Bill Of Materials (SBOM) for an image (Anchore Inc., 0.6.0)
  scan: Docker Scan (Docker Inc., v0.19.0)

Server:
 Containers: 18
  Running: 18
  Paused: 0
  Stopped: 0
 Images: 24
 Server Version: 24.0.7
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Using metacopy: false
  Native Overlay Diff: true
  userxattr: false
 Logging Driver: json-file
 Cgroup Driver: systemd
 Cgroup Version: 2
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: io.containerd.runc.v2 runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: 3dd1e886e55dd695541fdcd67420c2888645a495
 runc version: v1.1.10-0-g18a0cb0
 init version: de40ad0
 Security Options:
  apparmor
  seccomp
   Profile: builtin
  cgroupns
 Kernel Version: 5.15.0-89-generic
 Operating System: Ubuntu 22.04.3 LTS
 OSType: linux
 Architecture: x86_64
 CPUs: 4
 Total Memory: 7.635GiB
 Name: [server ID]
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false

</p>
</details> 

 - Browser: Chrome 120.0.6099.129 and Safari 17.2.1 on MacOS Sonoma 14.2.1 (23C71)

**Used app version**:
edge-debian-bullseye

**Comment by dude-at-RA** [comment]
Hey - I had the same issue trying to run 1.9.0 release but when I switched to 2.0.0 release it worked. I was running native with nodejs though, not in docker.

**Comment by drwb** [comment]
I'm running 2.0.0 as well. I've tried wiping everything, moving the underlying media to a new drive.

**Comment by rhatguy** [comment]
This same thing happened to me.  I thought something was wrong with my videos or something so I wiped the complete tmp directory before finding this.  I tried moving back to :latest and completely wiping everything, but it still seems broken for me.  I've started rebuilding all of my thumbnails and re-converting all of my video.  That will take a couple of days.

This is broken in the demo also so something has clearly gone wrong with the app
https://pigallery2.onrender.com/gallery?p=bunny.mp4

I'll try to do some more testing once I have everything rebuilt.
